.coafile

```diff
@@ -2,3 +2,7 @@
 bears = SpaceConsistencyBear
 files = .coafile
 use_spaces = True  
+
+
+
+   
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all`.*

The issue can be fixed by applying the following patch:

```diff
--- a/.coafile
+++ b/.coafile
@@ -5,4 +5,4 @@
 
 
 
-   
+
```